Stack/Opc.Ua.Core/Security/Certificates/CertificateFactory.cs

```diff
@@ -452,12 +449,13 @@ string password
     /// The issuer CA public key, the private key and the crl reside in the storepath.
     /// The CRL number is increased by one and existing CRL for the issuer are deleted from the store.
     /// </summary>
-    public static async Task RevokeCertificateAsync(
+    public static async Task<X509CRL> RevokeCertificateAsync(
```

Comment:
> Task<X509CRL> RevokeCertificateAsync( [](start = 23, length = 38)

split function in portion with store and with crl creation/signing

Reply:
future

---
In reply to: [155167313](https://github.com/OPCFoundation/UA-.NETStandard/pull/284#discussion_r155167313) [](ancestors = 155167313)